contents.txt

```diff
@@ -838,39 +809,8 @@
 ./sdk/6.0.100/de/Microsoft.TemplateEngine.Edge.resources.dll
 ./sdk/6.0.100/de/Microsoft.TemplateEngine.Orchestrator.RunnableProjects.resources.dll
 ./sdk/6.0.100/de/Microsoft.TemplateSearch.Common.resources.dll
-./sdk/6.0.100/de/Microsoft.TestPlatform.Build.resources.dll
-./sdk/6.0.100/de/Microsoft.TestPlatform.CommunicationUtilities.resources.dll
-./sdk/6.0.100/de/Microsoft.TestPlatform.CoreUtilities.resources.dll
-./sdk/6.0.100/de/Microsoft.TestPlatform.CrossPlatEngine.resources.dll
-./sdk/6.0.100/de/Microsoft.TestPlatform.Utilities.resources.dll
-./sdk/6.0.100/de/Microsoft.TestPlatform.VsTestConsole.TranslationLayer.resources.dll
-./sdk/6.0.100/de/Microsoft.VisualStudio.Coverage.IO.resources.dll
-./sdk/6.0.100/de/Microsoft.VisualStudio.TestPlatform.Client.resources.dll
-./sdk/6.0.100/de/Microsoft.VisualStudio.TestPlatform.Common.resources.dll
-./sdk/6.0.100/de/Microsoft.VisualStudio.TestPlatform.ObjectModel.resources.dll
 ./sdk/6.0.100/de/MSBuild.resources.dll
-./sdk/6.0.100/de/NuGet.Build.Tasks.Console.resources.dll
-./sdk/6.0.100/de/NuGet.Build.Tasks.resources.dll
```

Comment:
> [@omajid]
> For the localization stuff, do you have a plan to "fix" it so that source-build and Microsoft-built SDKs have the same resources (eg `-./sdk/6.0.100/de/NuGet.Build.Tasks.resources.dll`) at some point?